packages/ag-charts-website/src/content/docs/polar-axes/_examples/polar-axes-crosslines-angle/main.ts

```diff
@@ -0,0 +1,48 @@
+import { AgEnterpriseCharts, AgChartOptions } from 'ag-charts-enterprise';
+import { getData } from './data';
+
+const options: AgChartOptions = {
+    container: document.getElementById('myChart'),
+    data: getData(),
+    title: {
+        text: 'Skill Analysis',
+    },
+    series: [
+        {
+            type: 'radar-line',
+            angleKey: 'skill',
+            radiusKey: 'value',
+        },
+    ],
+    legend: {
+        enabled: true,
+    },
+    axes: [
+        {
+            type: 'angle-category',
+            shape: 'circle',
+            crossLines: [
+                {
+                    type: 'line',
+                    value: 'Meeting Deadlines',
+                    label: {
+                        text: 'Useless Skill',
```

Comment:
I think this might be somewhat controversial, some would argue being able to manage time, expectations and scope is a vital life skill :D